idb/postgres/internal/encoding/types.go

```diff
@@ -68,6 +68,7 @@ type evalDelta struct {
 	transactions.EvalDelta
 	GlobalDeltaOverride stateDelta            `codec:"gd"`
 	LocalDeltasOverride map[uint64]stateDelta `codec:"ld"`
+	LogsOverride        [][]byte              `codec:"lg"`
```

Comment:
We also need to add the following (plus encoding and decoding functions) in order to apply the existing conversions to inner transactions as well.

```go
InnerTxnsOverride []signedTxnWithAD `codec:"itx"`
```